src/operations.rs

```diff
@@ -71,12 +71,13 @@ pub fn download_extract_sans_parent(
             .progress_chars("=> "),
     );
 
-    let foo = pb.wrap_read(response);
+    let response_with_pb = pb.wrap_read(response);
 
-    let tar = GzDecoder::new(foo);
+    let tar = GzDecoder::new(response_with_pb);
```

Comment:
That is pretty cool, did it do this automatically?

Reply:
No that was me in the second commit ;-)